### ISSUE TITLE
support mermaid diagram blocks

### ISSUE THREAD

**Issue by erikerlandson** [planet42/Laika#331]
It would be cool to render mermaid diagrams, like github markdown:
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams#creating-mermaid-diagrams

**Comment by jenshalm** [comment]
Agreed, would be nice to have. It's too late for 0.19.0 which is nearing its completion, but I'll add it for the 0.19.1 milestone as there is probably a binary-compatible way to implement it.

**Comment by odenzo** [comment]
Or PlantUML since that may be more aligned as it has a Java Library

**Comment by jenshalm** [comment]
Completed in #516
